projects/spectator/src/lib/matchers.ts

```diff
@@ -247,7 +247,9 @@ export const toBeEmpty = comparator(el => {
  * 4. Type equal to "hidden" (only for form elements)
  * 5. A "hidden" attribute
  */
-function isHidden(el) {
+function isHidden(elOrSelector) {
+  let el = $(elOrSelector);
```

Comment:
If I remember correctly, this will not work as JQuery doesn't return the native element. We should obtain the first native element: 

```ts
let el = $(elOrSelector)[0]
if(!el) return undefined
```ts

Reply:
Yeah ;) hence closing the PR!

Reply:
Oh didn't notice :)